numba/errors.py

```diff
@@ -159,6 +172,31 @@ class ConstantInferenceError(NumbaError):
     """
 
 
+class InternalError(NumbaError):
+    """
+    For wrapping internal error occured with in the compiler
+    """
+    def __init__(self, exception):
+        super(InternalError, self).__init__(str(exception))
+        self.old_execption = exception
```

Comment:
Although it's not used, this might be a typo that will cause issues in future additions?